### ISSUE TITLE
Young Adult Onset Ages 

### ISSUE THREAD

**Issue by thePCCAS** [thePCCAS/onset_ages#1]
The Young Adult onset term in the HPO has a wide age range (16-40). Further segmenting this age range will improve pediatric curation and provide better representation for the Adolescent young adult population. The PCCAS is proposing to to further segment this range. The current proposed segmented ranges and terms are as follows:

16-20 – Adolescent Young Adult
21-30 – Early Young Adult 
31-40 – Late Young Adult

**Comment by thePCCAS** [comment]
Another potential set of ranges - 

16-19 – Adolescent Young Adult
20-29 – Early Young Adult
30-40 – Late Young Adult

**Comment by ckesserwan** [comment]
Given that we are adopting ICDO3.2, it would be helpful to retain the SEER's age categorization : https://seer.cancer.gov/report_to_nation/ 
Children: 0-14 yo
AYA: 15-39 yo

If we are to segment the AYA range, we need to provide supporting reference(s) to justify it. At this time, given that SEER and NCI did not further breakdown age categories, in my opinion, we should just continue to use the AYA 15-39 age range. Also narrowing the age range may bring more challenges and some confusion to the curators and users

**Comment by Laveniya** [comment]
Just a thought - could we educate the curators, when they sign up with CIViC, about age of onset? And run training for current curators?

Also, if a study includes multiple age ranges, e.g. 0 years to 18 years, do we have the option of selecting multiple age ranges?

**Comment by jsaliba10** [comment]
> Just a thought - could we educate the curators, when they sign up with CIViC, about age of onset? And run training for current curators?
> 
> Also, if a study includes multiple age ranges, e.g. 0 years to 18 years, do we have the option of selecting multiple age ranges?

Our decisions about the use of ages of onset will be ultimately included in the pediatric curation SOP. Also, we will be highlighting the ability to add onset to our training sessions. Once the additions of onset are piloted and discussed at the next few PCCAS meetings, Onset is something I would like to present in a more detailed manner to the members of the relevant taskforces.

Yes. In the associated phenotypes field you can enter multiple phenotypes and onsets. After adding one, you can click the green '+' to add another box.  In your example, my recommendation would be adding both "pediatric" (which covers up to 15 years) and young adult (which covers 16-40).

**Comment by jsaliba10** [comment]
@ckesserwan Young adult would remain the "parent" to the terms used to further break the age ranges up, similar to what is done in pediatric,  where pediatric is broken up further to juvenile and childhood. Curators would be asked to be granular with their selection of the age range.  Where do you see the challenge in curation for selecting an age range?

A few of the papers @Laveniya presented in May displayed an interesting shift of cancer types that occur as individuals age. These papers along with this [recommendation](https://www.liebertpub.com/doi/10.1089/jayao.2011.1505) could be the basis their inclusion.  Do you think there is value in separating the curations of those under 21 vs those in their 30s?

**Comment by ckesserwan** [comment]
@jsaliba10 , I fully agree, but in my opinion, further stratification needs to be based on an already established age-classification rendered by an authoritative entity such as SEER, CDC, AAP, etc... We have agreement regarding adopting the SEER age classification into 2 groups (Children: 0-14 yo, AYA: 15-39 yo) because this classification comes from SEER/WHO. But which reference you would enlist if you further stratify and adopt : 16-19 – Adolescent Young Adult, 20-29 – Early Young Adult, 30-40 – Late Young Adult ? I guess you could probably refer to PMID 32940362 from @Laveniya presentation ? or are you referring to https://www.liebertpub.com/doi/10.1089/jayao.2011.1505. I think as you long as you have a reference to justify your stratification, that should be fine !
